test/image/mocks/gl2d_selectedpoints.json

```diff
@@ -0,0 +1,14 @@
+{
+  "data": [{
+    "name": "array marker opacity edge case",
+    "type": "scattergl",
+    "mode": "markers",
+    "x": [1, 2, 3, 4, 5, 6],
+    "y": [1, 3, 2, 4, 5, 7],
+    "marker": {
+      "size": 20,
+      "opacity": [0.9, 0.8, 0.7, 1, 0.6, 0.8]
+    },
+    "selectedpoints": [1, 4, 2]
```

Comment:
On master, this gives

![image](https://user-images.githubusercontent.com/6675409/37978835-05ff9882-31b5-11e8-8eed-d79afde5d7e2.png)

where `selectedpoints` is essentially ignored.